lib/tic_tac_toe.ex

```diff
@@ -1,10 +1,22 @@
 defmodule TicTacToe do
   @moduledoc """
-  Documentation for `TicTacToe`.
+  The module that handles the logic and communcation for a Command Line Interface
+  Tic-Tac-Toe game
```

Comment:
```suggestion
  The module that handles the logic and communication for a Command Line Interface
  Tic-Tac-Toe game
```

Typo nitpick. Also I think since we don't depend on the `CommandLine` module directly and instead depend on the abstract `writer` that you could drop the "Command Line Interface" portion of this `moduledoc`. The only piece left in this module that really implies this is CLI TTT application are the contents of `@display_empty_board`.